### PR TITLE
Donate page

### LINKED ISSUE #224: Donate Page
## Issue Type:
- [ ] Bug Report 
- [x]  Feature Request 
- [ ] Documentaion

## **Describe the bug**

- Create a donate page

## **Possible solution**

- I would like to create a donate page for the foundation if you provide me the relevant details. 

## **Screenshots**

image

### PR DESCRIPTION
**Pull request by nidhivanjare**
# Description

Front end for donate page. 

Fixes #224

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Explain the Testing instructions

**Test Configuration**:
* Operating system: MAC OS
* Version: 11.2.3
* Text-editors used: vscode
 
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS

Screenshot 2021-05-01 at 3 10 59 PM

Screenshot 2021-05-01 at 3 11 11 PM

### CODE REVIEW

**Comment by rohank2502** [comment]
Please Deploy github pages  @nidhivanjare

**Comment by himanshu007-creator** [comment]
> Please Deploy github pages @nidhivanjare

yes @nidhivanjare , after that , it will be easier to work

**Comment by nidhivanjare** [comment]
https://nidhivanjare.github.io/SFC-foundations/

DONE!!

**Comment by rohank2502** [comment]
@himanshu007-creator  Please give it a look the Navbar is not like other pages should we go for this? And @nidhivanjare  I think we should add a link to every three projects because when we click project we get to head over to the home page and from that is user has to navigate own it on then we cant name the button projects we can rather add links to a page of each project in the name itself !!

**Comment by nidhivanjare** [comment]
I have added the project link 

Please let me know about the navbar though !!

**Comment by nidhivanjare** [comment]
@rohank2502. ??

**Comment by nidhivanjare** [comment]
@himanshu007-creator will you merge this brach or let me know any changes that I have to make ??

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator will you merge this branch or let me know any changes that I have to make ??

Please share the github page link. opened your latest deployment from <a href="https://github.com/nidhivanjare/SFC-foundations/deployments/activity_log?environment=github-pages">here</a> but the donate page showed error. Please look into it

File: index.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
                 <li><a href="#news">News</a></li>
                 <li><a href="pages/Contact-Us/contact.html">Contact us</a></li>
                 <li><a href="pages/Blog/blog.html">Blogs</a></li>
-                <li><a href="#">DONATE</a></li>
+                <li><a href="../../Assets/Donate.html">DONATE</a></li>
```
Comment (himanshu007-creator) [inline index.html:46]: <code>./Assets/Donate.html</code>
Try to change href with this and recheck all other links as well.

**Review by himanshu007-creator — CHANGES_REQUESTED** [review]

**Comment by nidhivanjare** [comment]
DONE!!

**Review by himanshu007-creator — CHANGES_REQUESTED** [review]
Its good to merge, just some minor changes

1) Favicon is not showing in the page. See other page for it.
2) the slideshow you created is probably broken, or if this is your idea , i dont get it.

you have done a good job. 1st problem is an easy fix, but the 2ns one needs some work. What i infer is that you want to make a slideshow. like this <a href="https://www.w3schools.com/w3css/w3css_slideshow.asp">example</a>. But its broken and thats why page behaves in a unusual way.

**Comment by nidhivanjare** [comment]
I will fix it and get back to you!!

**Comment by nidhivanjare** [comment]
Done @himanshu007-creator

**Comment by nidhivanjare** [comment]
Done @himanshu007-creator

Any other changes?

**Comment by nidhivanjare** [comment]
> Done @himanshu007-creator
> 
> Any other changes?

??

**Comment by himanshu007-creator** [comment]
> > Done @himanshu007-creator
> > Any other changes?
> 
> ??

nope, just resolve the conflicts

**Comment by himanshu007-creator** [comment]
> yes , i did just now

https://nidhivanjare.github.io/SFC-foundations/Assets/Donate.html#
still favicon is not there😬

**Comment by nidhivanjare** [comment]
Screenshot 2021-05-10 at 3 48 08 PM


You don't mean this ??

**Comment by nidhivanjare** [comment]
okay so I was using safari and there was no favicon there, I understood now !! 
will do right away

**Comment by nidhivanjare** [comment]
DONE

**Comment by himanshu007-creator** [comment]
Thanks a lot for your contribution @nidhivanjare 🎉✨

**Pull request merged**